modules/machine-health-checks-resource.adoc

```diff
@@ -39,9 +40,67 @@ format. For example, `prod-node-us-east-1a`.
 <4> Specify the timeout duration for a node condition. If a condition is met for the duration of the timeout, the Machine will be remediated. Long timeouts can result in long periods of downtime for the workload on the unhealthy Machine.
 <5> Specify the amount of unhealthy machines allowed in the targeted pool of
 machines. This can be set as a percentage or an integer.
+<6> Specify how long a MachineHealthCheck should wait for a Node to join the cluster,
+before considering a Machine unhealthy. This should be a duration string.
 
 [NOTE]
 ====
 The `matchLabels` are examples only; you must map your machine groups based on
 your specific needs.
 ====
+
+== Remediation short-circuiting
+
+To ensure that MachineHealthChecks only remediate Machines when the cluster is healthy,
+short-circuiting is implemented to prevent further remediation via the `maxUnhealthy`
+field within the MachineHealthCheck.
+
+If the user defines a value for the `maxUnhealthy` field
+(either an absolute number or a percentage of the total Machines checked by this MachineHealthCheck),
+before remediating any Machines, the MachineHealthCheck will compare the value of `maxUnhealthy`
+with the number of Machines within its target pool that it has determined to be unhealthy.
+If the number of unhealthy Machines exceeds the limit set by `maxUnhealthy`, remediation will *not* be performed.
+
+[IMPORTANT]
+====
+If `maxUnhealthy` is unset, it is assumed to mean `100%`.
+This means the short circuiting mechanism is *disabled by default* and Machines
+will be remediated no matter the state of the cluster.
+====
+
+=== Examples
+
+==== With an absolute value
+
+If `maxUnhealthy` is set to `2`:
+- If 2 or fewer nodes are unhealthy, remediation will be performed
+- If 3 or more nodes are unhealthy, remediation will not be performed
+
+These values are independent of how many Machines are being checked by the MachineHealthCheck.
+
+==== With percentages
+
+If `maxUnhealthy` is set to `40%` and there are 25 Machines being checked:
+- If 10 or fewer nodes are unhealthy, remediation will be performed
+- If 11 or more nodes are unhealthy, remediation will not be performed
```

Comment:
> If `maxUnhealthy` is set to `40%` and 25 Machines are checked:
> * Remediation is performed if 10 or fewer nodes are unhealthy.
> * Remediation is not performed if 11 or more nodes are unhealthy.

Use an `*` here instead of `-` for the list.